src/ArrayOfArraysView.hpp

```diff
@@ -618,6 +614,52 @@ class ArrayOfArraysView
     m_numArrays = newSize;
   }
 
+  /**
+   * @tparam POLICY The RAJA policy used to convert @p capacities into the offsets array.
+   *   Should NOT be a device policy.
+   * @brief Clears the array and creates a new array with the given number of sub-arrays.
+   * @param numSubArrays The new number of arrays.
+   * @param capacities A pointer to an array of length @p numSubArrays containing the capacity
+   *   of each new sub array.
+   * @param buffers A variadic pack of buffers to treat similarly to m_values.
+   */
+  template< typename POLICY, typename ... BUFFERS >
+  void resizeFromCapacities( INDEX_TYPE const numSubArrays,
```

Comment:
Should we disallow a device policy here?
```suggestion
  typename std::enable_if< POLICY != parallelDevicePolicy, void >::type
  resizeFromCapacities( INDEX_TYPE const numSubArrays,
```

Reply:
That would be nice but it's not that simple. For one the policy aliases in `LvArray` are only used in the testing, but more importantly there are multiple RAJA policies that wouldn't work here (anything that executes in another memory space) and `cuda_exec` is just one of those. I wouldn't be surprised if RAJA had some policy traits that told you this, but that's for another time.